dts/bindings/kscan/nuvoton,npcx-kscan.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX keyboard scan controller
+
+compatible: "nuvoton,npcx-kscan"
+
+include: [kscan.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+        required: true
+
+    clocks:
+        required: true
+
+    pinctrl-0:
+        required: true
+    pinctrl-names:
+        required: true
+
+    wui-maps:
+        type: phandles
+        required: true
+        description: |
+            Mapping table between Wake-Up Input (WUI) and KSIs.
```

Comment:
Could we use an interrupt-map for this?

https://devicetree-specification.readthedocs.io/en/latest/chapter2-devicetree-basics.html#interrupt-map

Reply:
The WUI (MIWU) is widely used in npcx drivers to generate interrupts for each group of 8 I/O ports.
If you think it's better to change it to interrupt-map, we might study how to achieve it and refactor all drivers.
The refactor will be done in another following PR as it is beyond the scope of the kscan driver.